### PR TITLE
[feat] Add ability to hide a batch of items in explorers

### PR DESCRIPTION
**Pull request by VkoHov**
- Added actions of batch hide and batch are shown in the table header
- Added batch hide to all explorers
- Tune hidden row styles

### CODE REVIEW

**Comment by VkoHov** [comment]
Resolves [#1923](https://github.com/aimhubio/aim/issues/1923)

File: aim/web/ui/src/components/Table/HideItemsModal.tsx [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+    data,
+  }: {
+    actionType: 'single' | 'selectAll' | 'removeAll';
+    data?: any;
+  }) => {
+    [key: string]: any;
+  };
+  onRowsVisibilityChange: (keys: string[]) => void;
+}): React.FunctionComponentElement<React.ReactNode> {
+  const actionText = hideMode ? 'hide' : 'show';
```
Comment (arsengit) [inline aim/web/ui/src/components/Table/HideItemsModal.tsx:38]: Let's move this to the bottom of the hooks

File: aim/web/ui/src/pages/Scatters/Scatters.tsx [hunk 2]
```diff
@@ -215,6 +215,8 @@ function Scatters(
                           archiveRuns={props.archiveRuns}
                           deleteRuns={props.deleteRuns}
                           focusedState={props.focusedState}
+                          onRowsVisibilityChange={props.onRowsVisibilityChange}
+                          visualizationElementType={'dots'}
```
Comment (arsengit) [inline aim/web/ui/src/pages/Scatters/Scatters.tsx:219]: Let's write enum for this

**Review by arsengit — CHANGES_REQUESTED** [review]

File: aim/web/ui/src/components/Table/DeleteModal.tsx [hunk 3]
```diff
@@ -15,16 +15,27 @@ import { processDurationTime } from 'utils/processDurationTime';
...
+    data?: any;
+  }) => {
+    [key: string]: any;
+  };
+  deleteRuns: (ids: string[]) => void;
+}): React.FunctionComponentElement<React.ReactNode> {
   const tableRef = React.useRef<any>({});
   const disabledTableRef = React.useRef<any>({});
-  const [dateNow, setDateNow] = React.useState(Date.now());
+  let runsDeleteRequest: any = null;
```
Comment (arsengit) [inline aim/web/ui/src/components/Table/DeleteModal.tsx:38]: I think the abort functionality isn't needed here. Mainly we use abort on getting requests.

**Review by arsengit — APPROVED** [review]

**Review by roubkar — APPROVED** [review]

**Pull request merged**